### PR TITLE
fix design

### PR DESCRIPTION
**Pull request by sustrishkenbrff**
I redid the design according to the one in the figma, downloaded the font, there is a problem with the shadow (I don't know how to fix it)

### CODE REVIEW

**Comment by q-13t** [comment]
1. remove background-image: url(/src/components/img/greenBackroundLoginPage.jpg); from .left-panel css
2. Add greenBackroundLoginPage.jpg into static files
3. Add  HTML element into <div className="left-panel">
4. Change display/positioning/z-index of  to display it under the "LOGO"
5. Add CSS to  element 
  1. Add transform in order to rotate and clip the image
  2. Add any drop-shadow CSS tags

POC:
image

**Comment by sustrishkenbrff** [comment]
@q-13t with your example, you gave me an idea of ​​how it can be fixed, reworked it a little differently

File: src/components/registrationPage/registrationPage.js [hunk 1]
```diff
@@ -63,6 +64,11 @@ const RegistrationPage = () => {
             </div>
             <div className="left-panel left-panel-reg">
                 <h1 className="logo">LOGO</h1>
+            <div className="img-wrap-right">
+                <div className="img-wraped-right">
+                    <img src={bgimg} alt="bg"/>
+                </div>
+            </div>
```
Comment (q-13t) [inline src/components/registrationPage/registrationPage.js]: Hawing img element wrapped in 2 divs is redundant. If we'll need to have them wrapped later we'll do it later
 ```suggestion
            <div className="img-wrap-right">
                    <img src={bgimg} alt="bg"/>
            </div>
```

File: src/components/loginPage/loginPage.js [hunk 2]
```diff
@@ -45,6 +46,12 @@ const LoginPage = () => {
             <div className="logo-main">
                 <h1 className="logo">LOGO</h1>
             </div>
+
+            <div className="img-wrap">
+                <div className="img-wraped">
+                    <img src={bgimg} alt="bg"/>
+                </div>
+            </div>
```
Comment (q-13t) [inline src/components/loginPage/loginPage.js]: Hawing img element wrapped in 2 divs is redundant. If we'll need to have them wrapped later we'll do it later
 ```suggestion
            <div className="img-wraper">
                    <img src={bgimg} className="img-register-bg" alt="bg"/>
            </div>
```

File: src/components/loginPage/loginPage.js [hunk 3]
```diff
@@ -45,6 +46,12 @@ const LoginPage = () => {
             <div className="logo-main">
                 <h1 className="logo">LOGO</h1>
```
Comment (q-13t) [inline src/components/loginPage/loginPage.js]:
```suggestion
                <h1 className="logo-register">LOGO</h1>
```

File: src/components/registrationPage/registrationPage.js [hunk 4]
```diff
@@ -63,6 +64,11 @@ const RegistrationPage = () => {
             </div>
             <div className="left-panel left-panel-reg">
                 <h1 className="logo">LOGO</h1>
```
Comment (q-13t) [inline src/components/registrationPage/registrationPage.js]:
```suggestion
                <h1 className="logo-login">LOGO</h1>
```

File: src/components/loginPage/loginPage.css [hunk 5]
```diff
@@ -15,17 +15,41 @@ body {
...
+    position: absolute; 
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0; 
+    z-index: -1;
+}
+
+.img-wraped-right {
+    clip-path: polygon(100% 0%, 0% 0%, 30% 100%, 100% 100%);
```
Comment (q-13t) [inline src/components/loginPage/loginPage.css]: No need to have a wrapper for each container and positioning for each element as the element occupies the whole page. NOTE: You may need to adjust the positioning/shadows/angles

```suggestion

.img-wrap {
    z-index: -1;
}

.img-register-bg {
    filter: drop-shadow(-30px 5px 20px rgba(0, 0, 0, 0.5));
    transform: translate(300px, 0px) scale(1.3) rotate(25deg);
}

.img-login-bg {
    filter: drop-shadow(30px 5px 20px rgba(0, 0, 0, 0.5));
    transform: translate(-300px, 0px) scale(1.3) rotate(-25deg);
}

.logo-register {
    position: absolute;
    top: 0px;
    left: calc(100% - 150px);
    /* TODO: make based on logo size */
    z-index: 1;
    margin: 15px;
    font-size: 2rem;
    font-weight: bold;
}

.logo-login {
    position: absolute;
    top: 0px;
    left: 0%;
    z-index: 1;
    margin: 15px;
    font-size: 2rem;
    font-weight: bold;
}
```

POC:
Before:
IQUBFQ2wVM
L6qUzCa7pW

After:
WpVWYMWI9c
VHPjrUmNAy

**Review by q-13t — COMMENTED** [review]
Give it a look, If necessary changes are needed, do them. I didn't do the positioning/scaling/rotating based on Figma template, so you'll need to change them as needed. Feel free to argue about any suggestions.

**Comment by q-13t** [comment]
If you'll find a way to make your code display correctly and be more responsive, feel free to do it your way, by try to get rid of double div — it's a bad practice.

**Review by q-13t — APPROVED** [review]

**Pull request merged**